test/Metadata.cpp

```diff
@@ -66,10 +66,13 @@ DEV_SIMPLE_EXCEPTION(CBORException);
 class TinyCBORParser
 {
 public:
-	explicit TinyCBORParser(bytes const& _metadata): m_pos(0), m_metadata(_metadata) {}
+	explicit TinyCBORParser(bytes const& _metadata): m_pos(0), m_metadata(_metadata)
+	{
+		assertThrow((_metadata.size() - m_pos) > 0, CBORException, "Input too short.");
```

Comment:
They are both unsigned, so probably `_metadata.size() < m_pos` would be better. But shouldn't it actually `m_pos < _metadata.size()`?

Reply:
Ok! Will change that and squash it into the right commit.